automation.py

```diff
@@ -55,6 +65,35 @@ def automate_passovers():
             print("Scheduled to automate at the next passover.")
     else:
         print("AUTOMATION: no more passovers found.")
+        # calculate new passovers for the next 24 hours using current TLE data
+
+        hk = housekeeping.get(local_args={'limit': 1, 'newest-first': 'true'})
+
+        tle = hk[0]['data']['logs'][0]['tle']
+        lines = tle.split('\n')
+    
+        orb = Orbital('ex-alta 2', line1=lines[0], line2=lines[1])
+        dtobj = datetime.utcnow()
+        passes = orb.get_next_passes(dtobj, 24, -113.4938, 53.5461, 0.645) # edmonton coordinates and elevation
+        ps_data = {'passovers': [{'timestamp': str(ps[0])} for ps in passes]}
+
+        passover.post(local_data=json.dumps(ps_data))
+        
+
+def send_slack_notifs(message):
```

Comment:
```suggestion
def send_slack_notifs(message):
    """Sends out a Slack message to all subscribed users.
    """
```
Add a doctoring here just in case we need to generate documentation in the future (and for readability purposes)

Reply:
Once this requested change is resolved this PR is good to merge